livekit-plugins/livekit-plugins-google/livekit/plugins/google/stt.py

```diff
@@ -303,10 +303,12 @@ async def input_generator():
                 retry_count = 0  # connection successful, reset retry count
 
                 await self._run_stream(stream)
-            except Aborted:
-                logger.error("google stt connection aborted")
-                break
-            except Exception as e:
+            except (Aborted, Exception) as e:
```

Comment:
I think we need to not retry or log errors if the input queue is closed.

Reply:
This will not be fired because `self._input_ch.closed` will stay false until agent session closes